docs/diagrams/AddOrderSequenceDiagram.puml

```diff
@@ -0,0 +1,90 @@
+@startuml
+!include style.puml
+skinparam ArrowFontStyle plain
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":AddOrderCommandParser" as AddOrderCommandParser LOGIC_COLOR
+participant "a:AddOrderCommand" as AddOrderCommand LOGIC_COLOR
+participant "r:CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Order ORDER_COLOR_T1
+participant ":Order" as Order ORDER_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant "m:Model" as Model MODEL_COLOR
+end box
+
+box Person PERSON_COLOR_T1
+participant ":Person" as Person PERSON_COLOR
+end box
+
+[-> LogicManager : execute(args)
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand(args)
+activate AddressBookParser
+
+create AddOrderCommandParser
+AddressBookParser -> AddOrderCommandParser
+activate AddOrderCommandParser
+
+create Order
+AddOrderCommandParser -> Order : new Order(orderArgs)
+activate Order
+
+Order --> AddOrderCommandParser : order
+deactivate Order
+
+AddOrderCommandParser --> AddressBookParser
+deactivate AddOrderCommandParser
+
+AddressBookParser -> AddOrderCommandParser : parse(args)
+activate AddOrderCommandParser
+
+create AddOrderCommand
+AddOrderCommandParser -> AddOrderCommand
+activate AddOrderCommand
+
+AddOrderCommand --> AddOrderCommandParser :
+deactivate AddOrderCommand
+
+AddOrderCommandParser --> AddressBookParser : a
+deactivate AddOrderCommandParser
+'Hidden arrow to position the destroy marker below the end of the activation bar.
+AddOrderCommandParser -[hidden]-> AddressBookParser
+destroy AddOrderCommandParser
+
+AddressBookParser --> LogicManager : a
+deactivate AddressBookParser
+
+LogicManager -> AddOrderCommand : execute(m)
+activate AddOrderCommand
+
+AddOrderCommand -> Model : getFilteredPersonList()
+activate Model
+
+Model --> AddOrderCommand : lastShownList
+
+AddOrderCommand --> Person : addOrder(order)
+activate Person
+
+Person --> AddOrderCommand :
+deactivate Person
+
```

Comment:
Should we also include the calls from AddOrderCommand to model.setPerson() and model.updateFilteredPersonList()?

Reply:
![image](https://github.com/AY2324S2-CS2103T-T16-3/tp/assets/76832850/cca85952-1def-49ef-bd6b-27851b577ec1)

I was looking at the developer guide at found that less important methods such as getFilteredPersonList are not necessary. So I kinda edited the diagram to include only the important methods. Not sure what you think?



![image](https://github.com/AY2324S2-CS2103T-T16-3/tp/assets/76832850/61f8bd46-8470-4831-a799-816ae10f9dea)